docs/source/cpp/index.rst

```diff
@@ -23,3 +23,4 @@ C++ Implementation
 
    getting_started
    api
+   examples
```

Comment:
The examples should come before the API IMHO.

Reply:
I'll let the CI finish, and if it passes I'll push with this tiny change.